buildscripts/BuildScripts.csproj

```diff
@@ -1,10 +1,34 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Before I do this are we sure we want to cause case clashes by using the name 'builddeps.csproj' instead of 'BuildScripts.csproj' in Solution Explorer?

![if-we-call-it-build-deps-aesthetics](https://cloud.githubusercontent.com/assets/607402/25638820/8b6f0e30-2f81-11e7-8651-bf31cda31923.gif)